packages/core/src/execution-engine/workflow-execute.ts

```diff
@@ -1077,10 +1092,16 @@ export class WorkflowExecute {
 				abortSignal,
 			);
 
-			const data =
-				nodeType instanceof Node
-					? await nodeType.execute(context)
-					: await nodeType.execute.call(context);
+			let data;
+
+			if (customOperation) {
+				data = await customOperation.call(context);
+			} else if (nodeType.execute) {
```

Comment:
So `customOperation` takes precedence over `.execute`? What it they are both defined? Is that possible?

Reply:
yes, it should take precedence, since it specifically defined for this resource and operation
yes, it's possible, but it's intended to be used with routing nodes that does not have execute method defined